src/pages/api/rss.ts

```diff
@@ -0,0 +1,23 @@
+import { aggs } from "@/core/elasticsearch";
+import { createReadStream } from "fs";
+import { NextApiRequest, NextApiResponse } from "next";
+import path from "path";
+
+export default async function docApiHandler(
```

Comment:
Look at other api routes, I added a Logger.

`export default LoggerApi(async function docApiHandler` (and import LoggerApi)